.github/workflows/assemble-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Assemble release and generate versions file
```

Comment:
Can we call the file something like `sync_release-manifest.yml`? :grin: and keep the workflow name a bit sorter as well, probably mirror it, i.e. "Sync Release Manifest"

Reply:
👍